wagtail/admin/templates/wagtailadmin/workflows/confirm_delete.html

```diff
@@ -6,12 +6,12 @@
     {% include "wagtailadmin/shared/header.html" with title=view.page_title subtitle=view.get_page_subtitle icon=view.header_icon %}
 
     <div class="nice-padding">
-        <p>This workflow is currently in progress on {{ workflow_states_in_progress }} pages. Deleting this workflow will cancel moderation on these page.
-            Are you sure you want to delete this workflow?</p>
+        <p>{% trans This workflow is currently in progress on {{ workflow_states_in_progress }} pages. Deleting this workflow will cancel moderation on these page.
+            Are you sure you want to delete this workflow? %}</p>
```

Comment:
Also. This new line will add unwanted whitespace to the translation text. Either use blocktrans or bring it back to one line.

Reply:
Yes i will be using blocktrans. I have written the corrected synatx in the comment above before i could push the code.